executor/executor.go

```diff
@@ -112,3 +158,10 @@ func execCommand(path string, args []string) (err error) {
 	lgr.Debug.Printf("System Time: %v, User Time: %v\n", state.SystemTime(), state.UserTime())
 	return
 }
+
+// CleanUp close file you use.
+func CleanUp() {
```

Comment:
Is it better to logger have a Close or Clean function?
```
package executor
lgr.Close() or lgr.Clean()

package logger
func (l *Logger)Close() {}
```

Reply:
Or when nil writeCloser is comming, logger set ioutil.Discard.
Then executor do not need check file is nil or not

Reply:
I fixed it.
I've decided to let logger have Close function.